src/ansys_sphinx_theme/assets/styles/pydata-sphinx-theme-custom.scss

```diff
@@ -779,3 +779,12 @@ button.btn.version-switcher__button:hover {
 span.math {
   display: math;
 }
+
+/**
+* code block name function
+*/
+
+div.highlight .nf{
+  font-family: var(--ast-body-family);
```

Comment:
We need a dedicated monospace font for code. So far, our body font is "Open Sans, sans-serif". Any chance we can have an `--ast-code-family`?

Reply:
That would be great!

Reply:
yeah sure . adding the monospace to code